client/httpClient.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"net/http"
+)
+
+var client *http.Client
```

Comment:
Can you explain why you need client to be package-global? Is it just for simplicity or anything more complex?

Reply:
I was thinking if the client ever was expanded in scope to allow for re-use, reduce connection establish time. I see your point though, usage of the CLI does not make this a great advantage.

Reply:
seeing your other comment, I can just be careful about reusing the client object instead of the global instance. This mitigates my concern.